src/components/AutoCompleteTextarea/Textarea.js

```diff
@@ -223,11 +223,7 @@ class ReactTextareaAutocomplete extends React.Component {
       /**
        * It's important to escape the currentTrigger char for chars like [, (,...
        */
-      new RegExp(
-        `\\${currentTrigger}${`[^\\${currentTrigger}${
-          trigger[currentTrigger].allowWhitespace ? '' : '\\s'
```

Comment:
remove these usage of `allowWhitespace ` and `afterWhitespace` options, since we never provide them in trigger settings